packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -254,7 +254,10 @@ export default class Import extends Component {
         }
       }),
       state.sr
-    )
+    ).then(res => {
+      return res
+    })
```

Comment:
Why?

Reply:
probably a cleanup issue there must have been a console.log() here during the development